libraries/AP_RCProtocol/AP_RCProtocol_CRSF.cpp

```diff
@@ -384,6 +384,7 @@ bool AP_RCProtocol_CRSF::decode_crsf_packet()
             process_link_stats_tx_frame((uint8_t*)&_frame.payload);
             break;
         default:
+            process_lua_pop();
```

Comment:
You need to get a custom frame ID registered with TBS rather than just assuming everything is LUA. I can do this for you if you would like.
Needs to be surrounded with #if AP_SCRIPTING_ENABLED

Reply:
Hi Andy,

Thanks for your prompt feedback. I've been using Ardupilot since APM2.5 with great joy, and it a pleasure to finally be able to make a small contribution.

My thought behind using 'default' instead of a specific custom frame ID was to give Lua the ability handle anything that is not handled by AP. Similar to what is happening on the EdgeTX side.

But there are of course also arguments that speak for using specific ID, for example to prevent a potential flooding of Lua with unwanted messages.

Let's go with the specific ID, I've added a placeholder for now: CRSF_FRAMETYPE_AP_LUA = 0x81.

Please get a custom frame ID registered with TBS for me. 

Cheers, Hugo

Reply:
Hugo, you can use 0x81 but please provide some kind of frame description so that this can be put into the CRSF specification. Thx Andy

Reply:
The frame is a standard (not extended) CRSF frame, without any data formatting for the payload. Any payload data formatting is handled by the AP Lua and transmitter Lua scripts. It is basically a transparent link, like the 3DR/SiK Radios, and could theoretically be used for MavLink...

CRSF Frame Format
```
uint8_t device_addr
uint8_t frame_length
uint8_t type = 0x81
uint8_t payload[1..60]
uint8_t crc
```
I fixed AP_RCProtocol_CRSF::process_lua_pop() to exclude crc.

I also raised a PR with ExpressLRS to include the 0x81 frame, see: https://github.com/ExpressLRS/ExpressLRS/pull/2062 . However, this AP PR will work also without the ELRS PR implemented, by setting the first payload byte to 0x00 in crossfireTelemetryPush() on the transmitter side.

I hope this PR is good to go now ;-)